PressureAdapter.java

```diff
@@ -0,0 +1,34 @@
+package sensor;
+
+import java.awt.Color;
+
+//Implements SensorInterface using PressureSensor methods
+public class PressureAdapter implements SensorInterface {
+	PressureSensor ps = new PressureSensor();
+	
+	@Override
+	public String getName() {
+		return ps.getSensorName();
+	}
+
+	@Override
+	public String getReport() {
+		return ps.getReport();
+	}
+
+	@Override
+	public double getValue() {
+		return ps.readValue();
+	}
+
+	//returns color based on report string
+	public Color getColor() {
+		String report = getReport();
```

Comment:
Also, the getReport() on this line should run your Overriden getReport() but it is a bit ambiguous because her class has a method of the same name. It would be safer to use _this.getReport()_